CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-Thanks for considering contributing to Default!
+Thanks for considering contributing to Hamagen - Israel's Ministry of Health COVID-19 exposure prevention app!
```

Comment:
Can you please change it to "Israel's Ministry of **Health's**"

Reply:
sure! I actually copy-pasted from the project's readme - so perhaps update that one as well :)